src/main/java/ru/spbstu/service/VariantService.java

```diff
@@ -0,0 +1,12 @@
+package ru.spbstu.service;
+
+import ru.spbstu.model.Variant;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public interface VariantService {
+
+    Iterable<Variant> save(InputStream inputStream, String dbName) throws IOException;
+    void saveWithBatch(InputStream inputStream, String dbName, int batchSize) throws IOException;
```

Comment:
Does the service client should really specify batchSize? If we're presenting a batch option, I think batch size should be provided via configuration or, not really cool though, via constant. What do you think?